e2e/app/page/notebook-cell.ts

```diff
@@ -0,0 +1,177 @@
+import {ElementHandle, Frame, Page} from 'puppeteer';
+import {getPropValue} from 'utils/element-utils';
+
+export enum CellType {
+  // To append to css selector
+  Code = '.code_cell',
+  Markdown = '.text_cell',
+  Any = '',
+}
+
+/**
+ * Notebook Cell represents the root element that contains both the code input and output cells.
+ */
+export default class NotebookCell {
+
+  private readonly page: Page;
+  private iframe: Frame; // Jupyter notebook iframe
+  private cellIndex: number; // 1-based cell index because notebook cell prompt index starts from 1.
+  private readonly cellType: CellType; // Markdown or Code cell.
+
+  /**
+   *
+   * @param {Page} page Puppeteer Page.
+   * @param {CellType} cellType Cell type: Code or Markdown cell. Default value is Code cell.
+   * @param {number} cellIndex 1-based cell index. Default value is 1.
+   */
+  constructor(page: Page, cellType: CellType = CellType.Code, cellIndex: number = 1) {
```

Comment:
My constructor comment I don't think was addressed: inline private here to avoid boilerplate?